src/main/java/frc/robot/commands/AutoCommand.java

```diff
@@ -35,9 +39,11 @@ public void execute() {
       stop();
       // also shoot here
     } else if (robotTimer.get() >= 4.5 && robotTimer.get() < 6) {
-      // shoot here
+      m_driveSubsystem.turnToAngle(m_gyroSubsystem.getYaw(), 90);
     } else if (robotTimer.get() >= 6 && robotTimer.get() < 10) {
+      m_driveSubsystem.resetPID();
       backward();
+      m_driveSubsystem.turnToAngle(m_gyroSubsystem.getYaw(), -90);
     } else {
       timer_complete = true;
```

Comment:
What are we trying to do here? Can we avoid having it be time based and instead use gyro readings to determine whether it's safe to advance to the next step?

Reply:
this is just an example for now, it will be replaced with something else soon.

Reply:
ok